Source/Core/DolphinWX/Debugger/MemoryView.cpp

```diff
@@ -161,7 +161,8 @@ wxString CMemoryView::ReadMemoryAsString(u32 address) const
     }
   }
 
-  return StrToWxStr(str);
+  // Not a UTF-8 string
+  return DataToWxStr(str);
```

Comment:
Please write `wxString(str.c_str(), wxCSConv(wxFONTENCODING_CP1252), str.size())` directly in here instead of making a function for it. Having a function for it obscures the fact that we've chosen to use Windows-1252, and having a function for it doesn't have any advantage like letting us cut down on repetition (since isn't really any other place we convert from Windows-1252 in DolphinWX).